src/components/header/navMenu.js

```diff
@@ -1,37 +1,56 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
+import { NavLink, withRouter } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import styles from './navMenu.module.scss';
 
-const NavMenu = ({ selectedTeam: { teamColor, urlName }, playerId }) => {
-  const activeStyle = {
-    color: teamColor,
+const NavMenu = ({
+  history: {
+    location: { pathname },
+  },
+  selectedTeam: { teamColor, urlName },
+  playerId,
+}) => {
+  const activeBorder = {
+    position: 'absolute',
+    width: 'calc(100% - 40px)',
+    left: '20px',
+    bottom: '0',
     borderBottom: `4px solid ${teamColor}`,
   };
 
   return (
     <div className={styles.container}>
       <div className={styles.linkContainer}>
-        <NavLink to={playerId ? `/${urlName}/players/${playerId}` : `/${urlName}/players`} activeStyle={activeStyle}>
+        <NavLink
+          to={playerId ? `/${urlName}/players/${playerId}` : `/${urlName}/players`}
+          activeStyle={{ color: teamColor }}>
           Players
+          <div style={pathname.indexOf('/players') >= 0 ? activeBorder : null} />
```

Comment:
Make this into a function call as it's being used elsewhere:

`  const getActiveStyle = (path) => (pathname.indexOf(path) >= 0 ? activeBorder : null);`

example:

`getActiveStyle('/players')`